exporter/observiqexporter/example/config.yaml

```diff
@@ -0,0 +1,20 @@
+receiver:
+  filelog:
```

Comment:
Typically, we want these example configs to use `nop` for all other components.

[For example](https://github.com/open-telemetry/opentelemetry-collector/blob/3f5c7180c51ed67a6f54158ede5e523822e9659e/exporter/otlpexporter/testdata/config.yaml#L5-L9).

Reply:
This one was intended to be an example file for how to use it, not necessarily test data. For instance, both the [splunk](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9c6bc6e00097d81ac59528d4b03fae88f26bb4f6/exporter/splunkhecexporter/example/otel-collector-config.yml) and [logzio](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9c6bc6e00097d81ac59528d4b03fae88f26bb4f6/exporter/logzioexporter/example/config.yaml) exporters have these example configurations, so my intention was to mirror that (although a lot of the exporters don't seem to include an example config, it seems).

I was planning on putting in that testdata config with the implementation, in a later PR. Do you think that that config is necessary for this first commit?

Reply:
Ah, my mistake. I didn't look closely enough and presumed that this was in the `testdata` directory. Looks good.